docs/getting-started/delta-lake-format.md

```diff
@@ -0,0 +1,185 @@
+
+:::info Docker on Windows
+You can use Docker Desktop for Windows together with Windows command line or Windows Linux Subsystem (WSL2) for this tutorial. But note that Docker Desktop for Windows needs a license for commercial use
+beginning of 2022.
+
+There is a free alternative for Linux or WSL2 called Podman from Redhat, which has a compatible command line and also the Dockerfiles are compatible, see [podman.io](https://podman.io/).
+Further advantages are that Podman is more lightweight - it doesn't need a service and root privileges to run containers.
+Install podman on WSL2 Ubuntu:
+
+    . /etc/os-release
+    echo "deb https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_${VERSION_ID}/ /" | sudo tee /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
+    curl -L "https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_${VERSION_ID}/Release.key" | sudo apt-key add -
+    sudo apt-get update
+    sudo apt-get -y upgrade
+    sudo apt-get -y install podman
+
+Install podman-compose for podman in WSL2:
+
+    sudo apt install python3-pip
+    sudo pip3 install podman-compose
+
+After starting `podman-compose up` in the getting-started folder you should now be able to open Polynote on port localhost:8192, as WSL2 automatically publishes all ports on Windows.
+If the port is not accessible, you can use `wsl hostname -I` on Windows command line to get the IP adress of WSL, and then access Polynote over {ip-address}:8192.
+:::
+
+But when you walk through the prepared notebook "SelectingData", you won't see any tables and data yet. 
+Can you guess why?  
+This is because your last pipeline run used an internal metastore, and not the external metastore we started with docker-compose yet.
+To configure Spark to use our external metastore, add the following spark properties to the application.conf under global.spark-options. 
+You probably don't have a global section in your application.conf yet, so here is the full block you need to add at the top of the file:
+
+    global {
+      spark-options {
+        "spark.hadoop.javax.jdo.option.ConnectionURL" = "jdbc:derby://metastore:1527/db;create=true"
+        "spark.hadoop.javax.jdo.option.ConnectionDriverName" = "org.apache.derby.jdbc.ClientDriver"
+        "spark.hadoop.javax.jdo.option.ConnectionUserName" = "sa"
+        "spark.hadoop.javax.jdo.option.ConnectionPassword" = "1234"
+      }
+    }
+
+This instructs Spark to use the external metastore you started with docker-compose. 
+Your Smart Data Lake container doesn't have access to the other containers just yet. 
+So when you run your data pipeline again, you need to add a parameter `--network getting-started_default` to join the virtual network where the metastore is located:
+
+    docker run --rm -v ${PWD}/data:/mnt/data -v ${PWD}/config:/mnt/config --network getting-started_default smart-data-lake/gs1:latest -c /mnt/config --feed-sel '.*'
```

Comment:
Using docker for windows, I still get an exception that it does not find the metastore:
`2021-12-11 11:14:55 ERROR Schema - Failed initialising database. [dag-1-59]
Unable to open a test connection to the given database. JDBC url = jdbc:derby://metastore:1527/db;create=true, username = sa. Terminating connection pool (set lazyInit to true if you expect to start your database after your app). Original Exception: ------
java.sql.SQLNonTransientConnectionException: java.net.UnknownHostException : Error connecting to server metastore on port 1,527 with message metastore.`

Reply:
I checked with Timo - Metastore container was not running because of some docker build problem (line-endings of scripts). I made an improvement for this in getting-started repo.